whetstone/build.gradle.kts

```diff
@@ -20,7 +20,7 @@ dependencies {
     implementation(libs.androidxCore)
     implementation(libs.androidxAppCompat)
     implementation(libs.androidxActivity)
-    implementation(libs.androidxFragment)
+    api(libs.androidxFragment)
```

Comment:
Do we want to make this optional in the near future?

Reply:
yeah, I think that might be worth it on the long run. @gent-ahmeti

Reply:
If you don't mind me chiming in on this one. I feel like optimally, this would be an add-on as the setup for compose is. And its usage could be the same `addOns.fragment.set(true)`.

Bringing in androidx fragments dependency transitively from my DI library would at least catch me off guard if anything. I understand this would probably mean quite a lot more work but just throwing it out there 😊 Thanks for reacting to my issue so fast!